index.js

```diff
@@ -9,4 +9,9 @@ const server = http.createServer((req, res) => {
 
 server.listen(port, () => {
   console.log(`Server running on http://localhost:${port}/`);
-});
\ No newline at end of file
+});
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing bracket `});` for the server.listen function is misplaced. It should be after the console.log statement.
```diff
- 12: \ No newline at end of file
- 13: });
+ 12:   console.log(`Server running on http://localhost:${port}/`);
+ 13: });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->